### PR TITLE
LOG-927: Update indexManagement script to use es-python client

### PR DESCRIPTION
**Pull request by sasagarw**
### Description

This PR:
- updates indexManagement curl commands to use es-python client.
- updates the referenced image for indexManagement to use curator image.

/cc @periklis 
/assign @igor-karpukhin 

### Links

- JIRA: https://issues.redhat.com/browse/LOG-927

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
In general looks good to me.

File: internal/indexmanagement/scripts.go [hunk 1]
```diff
@@ -1,5 +1,115 @@
...
+import os, sys
+import json
+from elasticsearch import Elasticsearch
+from ssl import create_default_context
+
+def getEsClient():
+  esService = os.environ['ES_SERVICE']
+  connectTimeout = os.getenv('CONNECT_TIMEOUT', 30)
+
+  tokenFile = open('/var/run/secrets/kubernetes.io/serviceaccount/token', 'r')
```
Comment (igor-karpukhin) [inline internal/indexmanagement/scripts.go:15]: `try...except` missing for `open` and `read`

Comment (sasagarw) [reply internal/indexmanagement/scripts.go:15]: @igor-karpukhin I had intentionally left that. Wherever we are calling this function, that is encapsulated between `try...except` hence any error would be caught and returned back.

**Pull request merged**